package.json

```diff
@@ -66,6 +66,8 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "mocha": "^5.0.0",
+    "mongojs": "^2.6.0",
+    "pg": "^7.4.3",
```

Comment:
Would you mind not removing this here? `pg` shouldn't be a dev dependency.

Reply:
removed both packages